logic/auth/MojangAccount.cpp

```diff
@@ -1,278 +0,0 @@
-		{
-			qWarning() << "Unable to load a profile because it was missing an ID or a name.";
-			continue;
-		}
-		profiles.append({id, name, legacy});
-	}
-
-	MojangAccountPtr account(new MojangAccount());
-	if (object.value("user").isObject())
-	{
-		User u;
-		QJsonObject userStructure = object.value("user").toObject();
-		u.id = userStructure.value("id").toString();
-		/*
-		QJsonObject propMap = userStructure.value("properties").toObject();
-		for(auto key: propMap.keys())
-		{
-			auto values = propMap.operator[](key).toArray();
-			for(auto value: values)
-				u.properties.insert(key, value.toString());
-		}
-		*/
-		account->m_user = u;
-	}
-	account->m_username = username;
-	account->m_clientToken = clientToken;
-	account->m_accessToken = accessToken;
-	account->m_profiles = profiles;
-
-	// Get the currently selected profile.
-	QString currentProfile = object.value("activeProfile").toString("");
-	if (!currentProfile.isEmpty())
-		account->setCurrentProfile(currentProfile);
-
-	return account;
-}
-
-MojangAccountPtr MojangAccount::createFromUsername(const QString &username)
-{
-	MojangAccountPtr account(new MojangAccount());
-	account->m_clientToken = QUuid::createUuid().toString().remove(QRegExp("[{}-]"));
-	account->m_username = username;
-	return account;
-}
-
-QJsonObject MojangAccount::saveToJson() const
-{
-	QJsonObject json;
-	json.insert("username", m_username);
-	json.insert("clientToken", m_clientToken);
-	json.insert("accessToken", m_accessToken);
-
-	QJsonArray profileArray;
-	for (AccountProfile profile : m_profiles)
-	{
-		QJsonObject profileObj;
-		profileObj.insert("id", profile.id);
-		profileObj.insert("name", profile.name);
-		profileObj.insert("legacy", profile.legacy);
-		profileArray.append(profileObj);
-	}
-	json.insert("profiles", profileArray);
-
-	QJsonObject userStructure;
-	{
-		userStructure.insert("id", m_user.id);
-		/*
-		QJsonObject userAttrs;
-		for(auto key: m_user.properties.keys())
-		{
-			auto array = QJsonArray::fromStringList(m_user.properties.values(key));
-			userAttrs.insert(key, array);
-		}
-		userStructure.insert("properties", userAttrs);
-		*/
-	}
-	json.insert("user", userStructure);
-
-	if (m_currentProfile != -1)
-		json.insert("activeProfile", currentProfile()->id);
-
-	return json;
-}
-
-bool MojangAccount::setCurrentProfile(const QString &profileId)
-{
-	for (int i = 0; i < m_profiles.length(); i++)
-	{
-		if (m_profiles[i].id == profileId)
-		{
-			m_currentProfile = i;
-			return true;
-		}
-	}
-	return false;
-}
-
-const AccountProfile *MojangAccount::currentProfile() const
-{
-	if (m_currentProfile == -1)
-		return nullptr;
-	return &m_profiles[m_currentProfile];
-}
-
-AccountStatus MojangAccount::accountStatus() const
-{
-	if (m_accessToken.isEmpty())
-		return NotVerified;
-	else
-		return Verified;
-}
-
-std::shared_ptr<YggdrasilTask> MojangAccount::login(AuthSessionPtr session,
-													QString password)
-{
-	Q_ASSERT(m_currentTask.get() == nullptr);
```

Comment:
I'm not entirely sure what you mean here since Github seems to have missed that this file moved...